File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/itembasket/Over.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.itembasket;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Over {

    @SerializedName("summ")
    @Expose
    private int summ;
    @SerializedName("summ_raw")
    @Expose
    private String summRaw;

    public int getSumm() {
        return summ;
    }

    public void setSumm(int summ) {
        this.summ = summ;
    }

    public String getSummRaw() {
        return summRaw;
    }

    public void setSummRaw(String summRaw) {
        this.summRaw = summRaw;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itembasket/Over.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itembasket

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Over {

    @SerializedName("summ")
    @Expose
    var summ: Int = 0
    @SerializedName("summ_raw")
    @Expose
    var summRaw: String? = null

}
